docs/index.rst

```diff
@@ -24,23 +24,17 @@ Table of Contents
 =================
 
 .. toctree::
-    :maxdepth: 4
-    :numbered:
```

Comment:
What was the reason for removing this - did it feel weird?

Reply:
Yeah, it didn't look right... However I reorganized the main sections and added them back; I think the numbered sections provide a feel of "moving forward" in terms of how we want people to read them, so we should definitely include them

Reply:
Looking at it, I think we should limit it to 2.